arangosh/Utils/ManagedDirectory.cpp

```diff
@@ -245,7 +245,7 @@ ManagedDirectory::ManagedDirectory(application_features::ApplicationServer& serv
       return;
     }
 
-    std::vector<std::string> files(TRI_FullTreeDirectory(_path.c_str()));
+    std::vector<std::string> files(TRI_FilesDirectory(_path.c_str()));
     bool isEmpty = (files.size() <= 1);
```

Comment:
The comparison is now wrong. In contrast to `TRI_FullTreeDirectory()`, `TRI_FilesDirectory()` does not return a vector containing the empty string as one of its elements.
```suggestion
    bool isEmpty = files.empty();
```